src/routes/namespaces/[namespace]/workflows/index.svelte

```diff
@@ -50,9 +51,9 @@
       {#if workflows.length}
         <WorkflowsSummaryTable>
           <tbody slot="rows">
-            {#each workflows as workflow}
-              <WorkflowsSummaryRow {workflow} {timeFormat} />
-            {/each}
+            <VirtualList items={workflows} let:item>
```

Comment:
This is kind of what we talked about the other day. I'm not convinced we can use a `<table>` for this view anymore. It the `VirtualList` might add something between the table heading and the table body that breaks the natural layout. I'm pretty sure you're going to have to fall back to an another element.